docs/content/docs/test/TestCaseInCode.md

```diff
@@ -15,6 +15,25 @@ type: docs
 
 Чтобы это сделать, тесту нужно проставить аннотацию `@ExternalId(UUID)`, где UUID - сгенерированный на клиенте случайный UUID.
 
+## Как происходит синхронизация
+
+[Report Viewer]({{< ref "/docs/test/ReportViewer.md" >}}) реализует механизм отправки событий.\
+Отчет помеченный как `finished` генерирует событие.\
+Сервис на стороне TMS слушает события, содержащие специальную json-метку, сообщающую что текущий отчет является источником правды\
```

Comment:
As I know all our reports marked `finished`
Is there additional logic on the TMS side? Report coordinates maybe?

Reply:
will be more specific (maybe draw a diagram)